content/blog/2021-08-31-aug-community-gems.md

```diff
@@ -0,0 +1,154 @@
+---
+title: August '21 Community Gems
+date: 2021-08-31
+description: |
+  A roundup of technical Q&A's from the DVC community.
+  This month: separate DVC pipelines, working with CML,
+  object storage as a data source, and more.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community.
+  This month: separate DVC pipelines, working with CML,
+  object storage as a data source, and more.
+picture: 2021-08-31/gems-cover.png
```

Comment:
we need to update it before we merge I think

Reply:
Yep! We're just waiting for Roman to send the image. :)

Reply:
Hi. So is this image the only thing pending? Should we then approve @shcheklein ? Thanks

Reply:
Compressed Image is up.  So I believe this is ready to go, though date is for tomorrow.  @flippedcoder @jorgeorpinel @shcheklein